catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -204,6 +235,53 @@ public static void addFileNodeWithAttributes(DirectedAcyclicGraph<Node, Edge> gr
             productLocal = false;
         }
         addBooleanAttribute(graph, fileNode, NsiliConstants.IS_PRODUCT_LOCAL, productLocal, orb);
+
+        if (metacard.getCreatedDate() != null) {
+            addDateAttribute(graph,
+                    fileNode,
+                    NsiliConstants.DATE_TIME_DECLARED,
+                    metacard.getCreatedDate(),
+                    orb);
+        } else {
+            addDateAttribute(graph, fileNode, NsiliConstants.DATE_TIME_DECLARED, new Date(), orb);
+        }
+    }
+
+    public static void addSecurityNodeWithAttributes(DirectedAcyclicGraph<Node, Edge> graph,
+            Node productNode, Metacard metacard, ORB orb) {
+        Any any = orb.create_any();
+        Node securityNode = new Node(0, NodeType.ENTITY_NODE, NsiliConstants.NSIL_SECURITY, any);
+        graph.addVertex(securityNode);
+        graph.addEdge(productNode, securityNode);
+
+        //TODO -- Once taxonomy is complete, set real values
+        addStringAttribute(graph,
+                securityNode,
+                NsiliConstants.CLASSIFICATION,
+                NsiliClassification.UNCLASSIFIED.getSpecName(),
+                orb);
+        addStringAttribute(graph, securityNode, NsiliConstants.POLICY, "NATO", orb);
```

Comment:
I'm going to leave this as it's temporary until we can map this data from the new taxonomy.